ui/app/mainui/AppMain.qml

```diff
@@ -918,6 +918,10 @@ Item {
                             systemPalette: appMain.sysPalette
                             emojiPopup: statusEmojiPopup
                         }
+                        onLoaded: {
+                            if (!item.profileContainer.currentItem)
+                                item.profileContainer.children[item.profileContainer.currentIndex].active = true
+                        }
```

Comment:
I don't like the how it's done here. `profileContainer` is exposed only because here we are accessing some internals to activate given loader. External context (AppMain in this case) has to know too much about those internals. What about doing it internally in `ProfileLayout` in `Component.onCompleted` to keep it hidden? And exposing  `profileContainer` is not needed then.

Reply:
Tried that (`Component.onCompleted`) and it didn't work (obviously), it's called only once and we want to perform the action every time the parent `Loader` is activated. But in general, I agree, I could expose a function to be called from outside.